### PR TITLE
Pipeline M21 14 - pipeline table

### PR DESCRIPTION
**Pull request by paulinakoz**
Cypress mochaweseome was required to pass the jenkins tests. The error message in the jenkins test said mochawesome in the cypress folder was missing.

### CODE REVIEW

**Comment by dearshrewdwit** [comment]
@paulinakoz Currently it's failing CI
- let's try a blunt tool: can you refresh your dependencies by removing the `package.lock` file and installing again by `npm install`

**Review by dearshrewdwit — CHANGES_REQUESTED** [review]
Great job passing CI!

The only things stopping this PR are
1.  the `.idea` directory - please add a commit removing the directory from version control
2. squash your commits!

File: cypress.json [hunk 1]
```diff
@@ -1,3 +1,4 @@
 {
-  "baseUrl": "http://localhost:3000"
+  "baseUrl": "http://localhost:3000",
+  "reporter": "cypress-mochawesome-reporter"
```
Comment (dearshrewdwit) [inline cypress.json:3]: what's this for?

Comment (sawaveys) [reply cypress.json:3]: We added cypress mochawesome because the jenkins test said it was missing

Comment (dearshrewdwit) [reply cypress.json:3]: Can you document that as part of the PR description?

**Review by dearshrewdwit — APPROVED** [review]
Nice. Let's get it in!

**Review by mattfann — APPROVED** [review]
All looks good - well done on getting through to a PR!

**Pull request merged**